socorro/external/es/connection_context.py

```diff
@@ -129,6 +130,12 @@ def indices_client(self, name=None):
             elasticsearch.client.IndicesClient(self.connection())
         )
 
+    def cat_client(self):
```

Comment:
What does this do? Can you add a docstring?

The elasticsearch-py docs aren't helpful at all:

https://elasticsearch-py.readthedocs.io/en/master/api.html?highlight=catclient#cat